src/shared/internal-calendar/types.ts

```diff
@@ -43,16 +43,19 @@ export interface InternalCalendarProps extends CommonCalendarProps {
     onSelect?: ((value: string) => void) | undefined;
     /** Called when day cell is hovered, returns value in `YYYY-MM-DD` */
     onHover?: ((value: string) => void) | undefined;
+    /** Indicate the number of days used in fixed-range variant */
+    numberOfDays?: number;
```

Comment:
since this is optional you should specify the `| undefined` per the repo conventions